setup-nexus-npm/README.md

```diff
@@ -1,22 +1,20 @@
 # setup-nexus-npm
 
-Configure a github action to point towards a self hosted Npm repository. After usage, any calls to `npm install` will point towards the repository at the endpoint provided.
+A composite github action for creating a .npmrc file that will point to the npm repositories in nexus
 
 ## Parameters
```

Comment:
I’m vacillating on getting rid of the endpoint flag.

We’ve changed the name of the Nexus endpoint several times, and in theory I like the possibility of having a build that targets nexus dev, even though that’s not possible at the moment.

I’d really like to reach a point where nexus dev is fully validated before it goes to prod, and I hate the amount of time we have to invest in it.